src/lib/organisms/Table/index.js

```diff
@@ -663,6 +663,7 @@ Table.propTypes = {
             copyable: PropTypes.bool,
             exportDisabled: PropTypes.bool,
             copyTooltipText: PropTypes.string,
+            copiedTooltipText: PropTypes.string,
```

Comment:
For `copyTooltipText` and `copiedTooltipText` doc comments are absent, let's add documentation comments for these props.